uw-frame-components/portal/widgets/partials/option-link.html

```diff
@@ -0,0 +1,51 @@
+<div class="widget-option-link" layout="column" layout-align="center center">
+
+  <!-- OPTION-LINK ICON -->
+  <div class="option-link-icon">
+    <a href="{{ widget.selectedUrl }}" target="_blank" rel="noopener noreferrer">
```

Comment:
Could this be more secure if it included [`rel="noopener"`](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)?